spec.md

```diff
@@ -387,16 +388,15 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 }' -X PUT -H "X-Broker-API-Version: 2.11" -H "Content-Type: application/json"
 ```
 
-### Response
-
 | Status Code | Description |
-| --- | --- |
-| 201 Created | Service instance has been provisioned. The expected response body is below. |
-| 200 OK | MAY be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
-| 202 Accepted | Service instance provisioning is in progress. This triggers the platform marketplace to poll the [Last Operation](#polling-last-operation) endpoint for operation status. |
+|---|---|
+| 200 OK | MUST be returned if the service instance already exists, is fully provisioned, and the requested parameters are identical to the existing service instance. The expected response body is below. |
```

Comment:
Since PUT is supposed to be idempotent, I was trying to distinguish between "already there", "completed as a result of this request" and "doing it asynchronously" as the platform may want to know what happened on this particular request.  IMO, the old wording implies what the new text says - just not as clearly.